library/spdm_common_lib/libspdm_com_context_data.c

```diff
@@ -48,11 +48,12 @@ libspdm_return_t libspdm_set_data(void *spdm_context, libspdm_data_type_t data_t
                                   size_t data_size)
 {
     libspdm_context_t *context;
-    uint32_t session_id;
+    uint32_t session_id, data32;
```

Comment:
Would you please split it to two line to make it clear?

```
uint32_t session_id;
uint32_t data32;
```

Reply:
Done. will be updated soon.